plugins/com.google.cloud.tools.eclipse.appengine.login/.classpath

```diff
@@ -3,5 +3,6 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
 	<classpathentry kind="con" path="org.eclipse.pde.core.requiredPlugins"/>
 	<classpathentry kind="src" path="src/"/>
+	<classpathentry kind="lib" path="lib/google-oauth-client-1.22.0.jar"/>
```

Comment:
The reason we can't use the one in maven central is that that jar does not have OSGI metadata, right?